jujugui/static/gui/src/app/app.js

```diff
@@ -729,6 +729,7 @@ YUI.add('juju-gui', function(Y) {
       var charmstore = this.get('charmstore');
       ReactDOM.render(
         <window.juju.components.UserProfile
+          authenticated={!!username}
```

Comment:
This will be incorrect once this lands: https://github.com/juju/juju-gui/pull/1410/files#diff-21dd467cb411bdd922556d1af7461864R728